package.xml

```diff
@@ -43,6 +43,7 @@
   <build_depend>rospy</build_depend>
   <run_depend>rospy</run_depend>
 
+  <run_depend>yaml</run_depend>
```

Comment:
Our installer can not handle this. Target should be ros-X. So please adjust this to python-pyyaml